db/migrate/20170512042542_create_categories.rb

```diff
@@ -0,0 +1,15 @@
+class CreateCategories < ActiveRecord::Migration
+  def change
+    create_table :categories do |t|
+      t.string :canonical_title, required: true
+      t.integer :anidb_id, index: true
+      t.string :image_content_type, limit: 255
+      t.string :image_file_name, limit: 255
+      t.integer :image_file_size
+      t.datetime :image_updated_at
```

Comment:
Does paperclip allow urls? There are some existing urls in the dump I was just adding to the file name field  if they were there, and leaving the rest empty.